src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1613,7 +1619,13 @@ void Convolution::initTryBrgconvFlag() {
             // should remove after binary postops performance issue resolved
             // heuristics: if it's  avx512 ISA model && it doesn't have binary post ops or per channel zero point.
             dnnl::primitive_attr attr;
-            setPostOps(attr, MemoryDescUtils::makeDummyShape(getOutputShapeAtPort(0)).getStaticDims(), false);
+            auto ouptuShape = getOutputShapeAtPort(0);
+            if (ouptuShape.isDynamic()) {
+                auto inpDummyShape = makeInputDummyShape(getInputShapeAtPort(0));
+                auto outputDims = shapeInferGeneric({ Shape(inpDummyShape), Shape(weightDims) });
+                ouptuShape = Shape(outputDims.front());
```

Comment:
Minor: seems lke it is better to extract the body into separate makeOutputDummyShape function and reuse within the node.